ee/tabby-db/migrations/0029_merged-provider-tables.up.sql

```diff
@@ -2,7 +2,7 @@ CREATE TABLE integration_access_tokens(
     id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
```

Comment:
Let's change the name of table to `integrations`.

Could be done in a follow up pr.